docs/implementationguidelines.md

```diff
@@ -0,0 +1,59 @@
+!!! info "Implementer guidance" 
+    This page contains guidance to implementers and is not part of the normative-track [FHIRcast specification](../specification/STU2).
+
+#Implementation Guide
+    
+## Single and Multiple Tab Applications
+Applications can have different capabilities and layouts, but with FHIRcast they should still be able to stay in sync. A potential situation that could cause confusion is when a single and a multi-tab application work together. 
+
+Let's start with a simple case.
+
+### Opening and Closing a Patient
+The diagrams below show that the apps will start without any context, then one will open a patient triggering a Patient-open event to the other app leading to that same patient being opened in the receiving app. When the patient is closed, that triggers a Patient-close event leading to the patient being closed in the other app as well.
+
+![Simple patient open and close example](/img/PatientOpenAndClose.png)
+
+
+### Opening Multiple Patients
+As you can see in the diagrams below, we can still maintain context sync between multiple and single-tabbed applications even when there is multiple contexts being opened. The initial Patient-open works as expected in syncing the two apps for that patient. When the multi-tab app opens another patient (without closing the first) the single-tab app should still follow the context change to keep the applications in sync. Even if the multi-tab app is leading the context changes, the single-tab app can still stay in sync with the current context.
+
+![Multiple patient open example](/img/MultiplePatientOpens.png)
+
+## Launching A Context-Less Tab
+Many applications can have a "home" or "default" tab that contains no clinical context, but may hold useful application features. In some cases other applications may want to subscribe to and be notified when another app has switched to the no context tab. To avoid confusion with other events, a new event will be created to represent a user switching to this context-less tab. Keep in mind that this is different than closing events and how it works between single and multi-tab applications should be considered. For example, similar to multiple open events (as described above) this event could be received, opening or switching to the context-less tab, without closing the currently open context. 
```

Comment:
```suggestion
Many applications can have a "home" or "default" tab that contains no clinical context, but may hold useful application features. In some cases other applications may want to subscribe to and be notified when another app has switched to the no context tab. To avoid confusion with other events, a new event will be created to represent a user switching to this context-less tab. Keep in mind that this is different than closing events and how it works between single and multi-tab applications should be considered. For example, similar to multiple open events (as described above) this event could be received, opening or switching to the context-less tab, without closing the previously open context. 
```
?